core/src/test/scala/org/http4s/rho/RhoRoutesSpec.scala

```diff
@@ -369,7 +369,7 @@ class RhoRoutesSpec extends Specification with RequestRunner {
         GET / "bar" |>> "bar"
       }
 
-      val routes2: HttpRoutes[IO] = "foo" /: routes1 toRoutes()
+      val routes2: HttpRoutes[IO] = "foo" /: routes1 toRoutes(i => i) // No idea why identity won't compile
```

Comment:
I think we need to look into why this won't compile a bit further before we commit this.  I worry that will cause users issues.

Reply:
For some reason Scala 2.13 interprets `something toRoutes()` as a attempt to pass `()` to the `toRoutes`. In other words it desugars to `something.toRoutes(())`. `something.toRequest()` works as expected. So I propose using `("foo" /: routes1).toRoutes()`.

Reply:
I'm fine with adding the parens.